crates/sui-framework/packages/deepbook/sources/clob_v2.move

```diff
@@ -1256,6 +1281,39 @@ module deepbook::clob_v2 {
         clock: &Clock,
         account_cap: &AccountCap,
         ctx: &mut TxContext
+    ): (u64, u64, bool, u64) {
+        place_limit_order_int(
+            pool,
+            client_order_id,
+            price,
+            quantity,
+            self_matching_prevention,
+            is_bid,
+            expire_timestamp, // Expiration timestamp in ms in absolute value inclusive.
+            restriction,
+            clock,
+            account_cap,
+            ctx
+        )
+    }
+
+    /// Place a limit order to the order book.
+    /// Returns (base quantity filled, quote quantity filled, whether a maker order is being placed, order id of the maker order).
+    /// When the limit order is not successfully placed, we return false to indicate that and also returns a meaningless order_id 0.
+    /// When the limit order is successfully placed, we return true to indicate that and also the corresponding order_id.
+    /// So please check that boolean value first before using the order id.
```

Comment:
I am not sure it makes sense to repeat the comment here and above

Reply:
removed in latest iteration.